pkg/dashboard/functiontemplates/render.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2017 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package functiontemplates
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/nuclio/nuclio/pkg/errors"
+	"github.com/nuclio/nuclio/pkg/functionconfig"
+
+	"github.com/ghodss/yaml"
+	"github.com/nuclio/nuclio-sdk-go"
+	"k8s.io/apimachinery/pkg/util/json"
+)
+
+type RenderConfig struct {
+	Template string                 `json:"template,omitempty"`
+	Values   map[string]interface{} `json:"values,omitempty"`
+}
+
+func Render(contents []byte) (*functionconfig.Config, error) {
```

Comment:
1. Should be an object, like all things
2. Should not do unmarshalling here - that's an interface thing. That should happen in the dashboard resource. Should accept a `RenderConfig`